query/src/main/scala/filodb/query/exec/RemoteExec.scala

```diff
@@ -0,0 +1,40 @@
+package filodb.query.exec;
+
+import monix.execution.Scheduler
+import monix.reactive.Observable
+import scala.concurrent.duration.FiniteDuration
+
+import filodb.core.{DatasetRef}
+import filodb.core.metadata.Dataset
+import filodb.core.query._
+import filodb.core.store.ChunkSource
+import filodb.query._
+
+
+final case class RemoteExecParams(logicalPlan: LogicalPlan, queryOptions: QueryOptions)
+
+final case class RemoteExec(id: String,
+                      dispatcher: PlanDispatcher,
+                      dataset: DatasetRef, params: RemoteExecParams,
```

Comment:
Why RemoteExecParams ?
I am thinking it would be simple to add the logicalPlan and queryOptions in the RemoteExec itself.

Reply:
Oh Never mind. I get it. RemoteExecParams is the one sent out in serialized form.